src/transformers/modeling_utils.py

```diff
@@ -812,7 +812,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
             output_loading_info(:obj:`bool`, `optional`, defaults to :obj:`False`):
                 Whether ot not to also return a dictionary containing missing keys, unexpected keys and error messages.
             local_files_only(:obj:`bool`, `optional`, defaults to :obj:`False`):
-                Whether or not to only look at local files (e.g., not try doanloading the model).
+                Whether or not to only look at local files (i.e. do not try to download the model).
```

Comment:
```suggestion
                Whether or not to only look at local files (i.e., do not try to download the model).
```
Super nit, but O'Reilly made me had those commas after i.e. and e.g. everywhere, so now I like them :-)